Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+def dockerImage = 'build-tools/android-build-box:latest'
```

Comment:
why not library?

Reply:
We don't have jenkins library for android libraries. We don't have rules, agreements, reference project (gradle helper) for it. We don't have it for java libraries neither.